pyquil/operator_estimation.py

```diff
@@ -814,11 +815,71 @@ def serializable(self):
         }
 
 
+def generate_experiment_programs(tomo_experiment: TomographyExperiment,
```

Comment:
let's keep these functions private for now -- i'd like to discuss the overall API in more detail, but I don't want to block progress on improving the organization / readability of this file